build/build_cross-platform.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+binary="portainer-$1-$2"
+
+docker run -tv $(pwd)/api:/src -e BUILD_GOOS="$1" -e BUILD_GOARCH="$2" portainer/golang-builder:cross-platform /src/cmd/portainer
+
+mkdir -p dist
+mv "api/cmd/portainer/$binary" dist/
+shasum "dist/$binary" > portainer-checksum.txt
```

Comment:
Indeed, in alpine I use `sha256sum`. It is a little trickier because shasum exists, so a ln is not possible.
I think that `sha256sum` is portable enough. However, since it is not used, I commented it. I didn't remove it, because it is a single line and it may be useful in the near future.